packages/sdk/src/libs/clients/erc20/client.test.ts

```diff
@@ -0,0 +1,26 @@
+require("dotenv").config();
+import assert from "assert";
+import * as Client from "./client";
+import { ethers } from "ethers";
+
+const address = "0xeca82185adCE47f39c684352B0439f030f860318";
+// these require integration testing, skip for ci
+describe.skip("erc20", function () {
```

Comment:
would it be better to not use the sky syntax and have a different way to toggle if tests are run or not?

Reply:
the right thing to do here is probably to add hard hat tests, but i've been kicking that can down the road. if you think that is probably the right course of action here i will make an issue for it

Reply:
added a low priority card for this on clubhouse